.github/workflows/pr-sanity.yaml

```diff
@@ -22,21 +22,21 @@ jobs:
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - uses: actions/setup-go@v2
+      - uses: actions/setup-go@v3
         with:
-         go-version: '1.19' # The Go version to download (if necessary) and use.
+          go-version: '1.19' # The Go version to download (if necessary) and use.
 
       - name: Do sanity checks
         run: make sanity
 
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v2.5.2
+        uses: golangci/golangci-lint-action@v3
         with:
-          skip-go-installation: true
```

Comment:
Why skip go installation? we are doing it in line 27

Reply:
see: https://github.com/golangci/golangci-lint-action#compatibility
```
v3.0.0+ requires explicit setup-go installation step prior to using this action: uses: actions/setup-go@v3. The skip-go-installation option has been removed.
```